templates/part-category/part-category.js

```diff
@@ -126,10 +151,26 @@ export default async function decorate(doc) {
   const breadcrumbBlock = main.querySelector('.breadcrumb-container .breadcrumb');
   const titleWrapper = createElement('div', { classes: 'title-wrapper' });
   const title = createElement('h1', { classes: 'part-category-title' });
+
+  titleWrapper.appendChild(title);
```

Comment:
have you considered to set this as a function and add it to titleWrapper just above of the section injection to avoid the content shift?